SAP/spartacus
# Feature/cxint 2989 download proposal

https://jira.tools.sap/browse/CXSPA-6886

The following is a checklist for new epics or features acceptance for Spartacus, based on [our Definition of Done document](https://sap.github.io/spartacus-docs/definition-of-done/)

## General

- [x] Epic has been tested on a production-like server using Spartacus installation script
- [x] Epic has been tested in all supported browsers and devices (Check list on DoD page)
- [x] Epic has sufficient end to end (cypress) test coverage
- [ ] Schematics code deprecations have been provided and tested
- [x] Epic complies with [Security best practices](https://sap.github.io/spartacus-docs/security-best-practices/)
- [x] Epic complies with [Accessibility best practices](https://sap.github.io/spartacus-docs/a11y-best-practices/)
- [x] Epic CSS supports directionality (if required)
- [ ] Epic code scans have been conducted [Compliance scans for branches](https://wiki.one.int.sap/wiki/display/spar/Compliance+scans+for+branches)
- [ ] Epic renders successfully in SSR

## Audits/reviews

- [ ] Architecture review of the epic
- [ ] Security and threat modeling audit/review from the Security team
- [ ] Accessibility audit/review from the UX team

## New Library

If your epic will be introduced or introduces a new library:

- [ ] Library provides schematics to install the library
- [ ] Library modules can be lazy loaded appropriately
- [ ] Library has been tested for a customer using Spartacus installation script

## Sample data

If the new feature requires new or updated sample data for a specific Commerce backend version:

- [ ] Sample data has been added to the sampledata repo and merged to the next version to be released
- [ ] New sample data has been loaded and tested on a dev server
- [ ] New sample data has been loaded and tested in our CI server

## Documentation

- [ ] Documentation input is provided for it to be published on the doc site.

## How to test
- change your base url to s1 api url (https://api.cg79x9wuu9-eccommerc1-s1-public.model-t.myhybris.cloud/)
- start the server
- login with user ID as **james.weber@harvestlive.inc** and password as **welcome**
- go to quote details for Quote ID 02791913
- download link is available besides the links for **New Cart** and **Quotes**
- click on the link and check that the file is downloaded
